.meteor/packages

```diff
@@ -17,23 +17,15 @@ es5-shim@4.8.0
 
 # Collections
 aldeed:collection2
-wekan-cfs-standard-packages
+cfs:standard-packages
 cottz:publish-relations
 dburles:collection-helpers
 idmontie:migrations
-matb33:collection-hooks
 matteodem:easy-search
-mongo@1.11.0
+mongo@1.14.0
 mquandalle:collection-mutations
 
 # Account system
-kenton:accounts-sandstorm
-service-configuration@1.0.11
-useraccounts:unstyled
-useraccounts:flow-routing
-wekan-ldap
-wekan-accounts-cas
-wekan-accounts-oidc
```

Comment:
wekan-accounts-oidc has support for many custom OIDC/OAuth2 login types, including:
- https://github.com/wekan/wekan/wiki/Oracle-OIM
- https://github.com/wekan/wekan/wiki/ADFS
- https://github.com/wekan/wekan/wiki/OAuth2#auth

Is there some real working replacement available somewhere?